projects/js-packages/licensing/components/activation-screen/utils.js

```diff
@@ -1,3 +1,9 @@
+const JETPACK_PLUGIN_FILE_NAME = 'jetpack/jetpack.php';
+const JETPACK_PLUGIN_DEV_FILE_NAME = 'jetpack-dev/jetpack.php';
+const JETPACK_SOCIAL_PLUGIN_FILE_NAME = 'jetpack-social/jetpack-social.php';
+const JETPACK_SOCIAL_PLUGIN_DEV_FILE_NAME = 'jetpack-social-dev/jetpack-social.php';
+const JETPACK_SOCIAL_PLUGIN_LOCAL_FILE_NAME = 'social/jetpack-social.php';
```

Comment:
I got these values by examining the file names on local, JN and my pressable site, which would be the equivalent of a hosted site that users would have.